applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -28,6 +28,21 @@ config BLE_LE_POWER_CONTROL_ENABLED
 	  dynamically and automatically during connection, which provides effective
 	  communication.
 
+config BT_DEVICE_NAME_ALT
+	string "Alternative Bluetooth device name"
+	default "NRF5340_AUDIO_ALT"
+	depends on TRANSPORT_BIS
+	help
+	  Alternative Bluetooth device name. Name can be up to 248 bytes long
+	  (excluding NULL termination). Can be empty string.
+
+config BT_ALTERNATIVE_DEVICE_NAME
```

Comment:
Not sure it would work as the name is tested to check if it is the one we are interested in at the headset. Thus it may keep using one gateway and not switch, it's operation will be indeterministic. I therefore need a seconded alternative name to the default we use. Correct on programming two gateways separately and I was using the alternative name bool to do this without the user having to remember to rename the gateway before a compile. Which should help prevent name mismatches. Also the name needs to be in the list we are looking for so would need to have both defined for the headset. 

So I believe this is useful and needed.

Reply:
I kept this functionality